doc/source/reference/storage/cinder-guide-lightbits.rst

```diff
@@ -0,0 +1,33 @@
+.. cinder-guide-Lightbits:
+
+=====================================
+Lightbits labs storage for OpenStack
+=====================================
+
+Lightbits labs Cinder Driver
```

Comment:
same and same everyone you have "labs"

Reply:
@yuval-lb  Will this change the content of this page ?
  https://docs.openstack.org/cinder/latest/configuration/block-storage/drivers/lightbits-lightos-driver.html

I've plan to create patch to update these pages .

Reply:
if yes then we want to merge this patch asap.

Reply:
@rahman-lb no, this guide will be part of the kolla-ansible project

Reply:
Got you, i'll take care of the other.